get_dependencies.py

```diff
@@ -23,19 +23,39 @@ def download(url, filename):
                 sys.stdout.flush()
     sys.stdout.write('\n')
 
-
-print('[*] Extracting webdriver archives...')
-for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
-    tar = tarfile.open('tools/%s.tar.xz' % f, 'r:xz')
-    tar.extractall(path='tools/')
-    tar.close()
+if sys.platform == "linux":
+    print('[*] Downloading support files for linux ~ 108 MB')
+    download('https://www.dropbox.com/s/3mosdy9tsf7pilh/linux.zip?dl=1', 'linux.zip')
+    print('[*] Extracting linux.zip...')
+    with ZipFile('linux.zip', 'r') as z:
+        z.extractall("linux")
+    print('[*] Extracting webdriver archives...')
+    for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
```

Comment:
You have removed these files from the repository, so this code should be removed.

Reply:
actually earlier files are compressed directly and when they are downloaded they get in this form only

Reply:
There's no need to compress them twice then 😄 
Just compress all of the extracted files together in a tar.xz archive.

Reply:
okay